pkgs/development/python-modules/moto/default.nix

```diff
@@ -88,9 +88,8 @@ buildPythonPackage rec {
               -e test_passthrough_requests
   '';
 
-  # Disabling because of 20 failing tests due to https://github.com/spulec/moto/issues/2728
-  # We should enable these as soon as possible again though. Note the issue
-  # is unrelated to the docutils 0.16 bump.
+  # Disabling because multiple tests still rely on boto, like
```

Comment:
Would it be an option to put `boto` only in the checkInputs and enable the `doCheck`?

Reply:
Ah. You already mentioned that boto is broken on python3.9, so that wouldn't work there.

Reply:
Maybe there's a subset of the tests that could be enabled. I'll look at that. Or run doCheck = pythonOlder "3.9"; would be another option.

Reply:
boto is not supported with 3.9 because it is dead and has been replaced by boto3

Reply:
Even after the merge of `https://github.com/spulec/moto/pull/3503` upstream, many of the tests still `import boto` which is a bit unfortunate. I'll try to find at least a few that don't so we can enable a couple tests here.